### DISCUSSION TITLE
Add location multiple times for events.

### DISCUSSION THREAD

**Discussion by shushyz** [jynnie/soullocke#4] [Ideas]
Think there should be an options to reuse/add used locations if needed. Think this would be useful for scenarios for like npc trades, egg hatchings, and the likes.

**Comment by shushyz** [comment]
This would also be useful for those who play with static encounters (legendaries, event pokemon, shiny).

**Comment by jynnie** [comment]
That makes sense. I'll look into this 🤔

**Comment by jynnie** [comment]
Just implemented! Let me know if there's any issues you run into.
